doc/developer/scratch.md

```diff
@@ -124,5 +126,87 @@ bin/scratch destroy --all-mine
 Pass `--dry-run` if you want to see what instances `bin/scratch destroy` would
 destroy without actually destroyin them.
 
+### Grafana integration
+
+The `materialized` process always exposes metrics at via its primary port's http server on the
+prometheus-standard `/metrics` path, but the deployed server needs to be configured correctly for
+our Prometheus server to actually scrape the metrics and thereby expose them to Grafana. You
+
+The tl;dr is that you must configure mzcompose with `--preserve-ports` and the EC2 instance with
+the `"scrape_benchmark_numbers": "true"` tag. Thus a bare-minimum grafana-integrate config looks
+like:
+
+```javascript
+{
+    "launch_script": "bin/mzcompose --preserve-ports ..<remainder of args>"
+    // .. snip config ..
+    "tags": {
+        "scrape_benchmark_numbers": "true"
+    }
+}
+```
+
+Read on for more details and some other items that can be configured.
+
+#### Prometheus config
+
+There are three tags that configure our Prometheus' integration, and if you are running inside
+mzcompose it must also be configured correctly.
+
+The three Prometheus tags are:
+
+* `scrape_benchmark_numbers`: must be set to exactly the string `"true"` in order for Prometheus
+  to observe the instance.
+* `purpose`: is used as a filter in the Grafana UI. You can use this to group all your instances
+  (e.g. set it to `myname-debugging`) or set it to `load-test` or `benchmark`.
+* `test` is both used by scratch to determine which test to execute, and displays as an additional
+  filter inside of the Grafana UI.
+
+So the minimum scratch config to get metrics into Grafana looks like:
+
+```javascript
+{
+    // .. snip general config ..
+    "tags": {
+        "scrape_benchmark_numbers": "true"
+    }
+}
+```
+
+And a slightly more complete one could be:
+
+```javascript
+{
+    // .. snip general config ..
+    "tags": {
+        "scrape_benchmark_numbers": "true",
+        "purpose": "bwm-debugging",
+        "test": "chbench"
+    }
+}
+
+{
+    // .. snip general config ..
+    "tags": {
+        "scrape_benchmark_numbers": "true",
+        "purpose": "bwm-debugging",
+        "test": "billing"
+    }
+}
+```
+
+#### MzCompose config
+
+Prometheus only looks for materialize metrics on port 6875. The canonical way to ensure that
+Materialize is available on port 6875 on a host is to pass the `--preserve-ports` argument to
+mzcompose.
```

Comment:
```suggestion
Prometheus only looks for Materialize metrics on port 6875. The canonical way to ensure that
Materialize is available on port 6875 on a host is to pass the `--preserve-ports` argument to
mzcompose. (Without this flag, mzcompose chooses a random host port for Materialize, which
will be unknown to Prometheus.)
```

(Just added a small note about what goes wrong if you don't specify `--preserve-ports`.)